WeRadiate-ThermoSense.ino

```diff
@@ -289,17 +291,35 @@ void setup_platform(void)
                 flags = 0;
                 }
 
+	/* initialize the FLASH */
+	if (gFlash.begin(&gSPI2, Catena::PIN_SPI2_FLASH_SS))
+		{
+		fFlash = true;
+		gFlash.powerDown();
+		gCatena.SafePrintf("FLASH found, put power down\n");
+		}
+	else
+		{
+		fFlash = false;
+		gFlash.end();
+		gSPI2.end();
+		gCatena.SafePrintf("No FLASH found: check wiring\n");
+		}
+
         /* is it modded? */
         uint32_t modnumber = gCatena.PlatformFlags_GetModNumber(flags);
 
-        //fHasPower1 = false;
+        fHasPower1 = false;
 
         if (modnumber != 0)
                 {
                 gCatena.SafePrintf("Catena 4612-M%u\n", modnumber);
                 if (modnumber == 102 || modnumber == 103 || modnumber == 104)
                         {
                         fHasCompostTemp = flags & CatenaBase::fHasWaterOneWire;
+                        /* set D11 high so V_OUT2 is going to be high for onewire sensor */        
```

Comment:
This seems to duplicate code below. Since you check every time, maybe this isn't needed.